packages/design-system-tokens/src/lib/file.ts

```diff
@@ -72,6 +72,7 @@ export const getFileDescriptors = (rootPath: string): FileDescriptor[] => {
 export const writeFile = (filename: string, fileData: string): number => {
   try {
     fs.writeFileSync(filename, fileData);
+    // eslint-disable-next-line no-console
     console.log(`:: wrote ${filename}`);
```

Comment:
I wasn't sure if this was spurious or not, so I left it and disabled the error. Glad to remove it if that's the right thing.

Reply:
Let's keep it 👍